### ISSUE TITLE
[12.0] access_apps: can't restrict installing apps for users

### ISSUE THREAD

**Issue by trojikman** [it-projects-llc/access-addons#195]
As stated in documentation ( https://github.com/it-projects-llc/access-addons/blob/12.0/access_apps/doc/index.rst ) if you choose to Select blank line in ``Apps access`` security category, then User can't install apps from anywhere. But, if we go to module's ``Configuration`` (e.g Website's configuration), we can see ``Apps`` menu item which in my opinion is wrong.

**Comment by yelizariev** [comment]
You can see, but you cannot use

**Comment by yelizariev** [comment]
Send a screenshot, please

**Comment by trojikman** [comment]
![user_settings](https://user-images.githubusercontent.com/23641186/72055429-a3cccc80-32ec-11ea-93b4-ffca26730a5f.png)
![2](https://user-images.githubusercontent.com/23641186/72055467-b3e4ac00-32ec-11ea-8c35-a48a7bdeefc6.png)
![3](https://user-images.githubusercontent.com/23641186/72055480-ba732380-32ec-11ea-9ea7-a69ae276adda.png)

**Comment by trojikman** [comment]
> You can see, but you cannot use

Yes, I can't install module, because it throws a server error, but should it work this way?

**Comment by yelizariev** [comment]
It's a bug. That menu should be hidden in a new module. Say ``access_apps_website``

cc @ilmir-k

**Comment by trojikman** [comment]
> It's a bug. That menu should be hidden in a new module. Say `access_apps_website`
> 
> cc @ilmir-k
Are you talking about this?
https://github.com/it-projects-llc/access-addons/tree/12.0/access_apps_website

**Comment by yelizariev** [comment]
Да. Я сначала вспомнила есть ли уже такой модуль, но не нашел. 
Его запланировали удалить, но всё таки нужен
